Description: Remove the dependencies from setup.py to prevent issues with pkg_resources.
 Even though the dependencies are installed, pkg_resources won't find them. Since these
 lists aren't used anywhere in the actual program I removed them to fix this issue.
 .
 knossos (0.2.0-1) xenial; urgency=medium
 .
   * New upstream release
Author: ngld <ngld@tproxy.de>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

Index: knossos-0.4.0/setup.py
===================================================================
--- knossos-0.4.0.orig/setup.py
+++ knossos-0.4.0/setup.py
@@ -61,7 +61,7 @@ setup(
 
     keywords='fso freespace',
     packages=['knossos', 'knossos.ui', 'knossos.third_party'],
-    install_requires=['six', 'requests', 'semantic_version', 'raven', 'PyQt5'],
+    install_requires=[],
 
     # List additional groups of dependencies here (e.g. development dependencies).
     # You can install these using the following syntax, for example:
